python_modules/dagster/dagster_tests/daemon_sensor_tests/test_run_status_sensors.py

```diff
@@ -42,31 +43,37 @@ def instance_with_sensors(overrides=None, attribute="the_repo"):
         with create_test_daemon_workspace_context(
             workspace_load_target(attribute=attribute), instance=instance
         ) as workspace_context:
-            yield (
-                instance,
-                workspace_context,
+            repository_location = check.not_none(
                 next(
                     iter(
                         workspace_context.create_request_context().get_workspace_snapshot().values()
                     )
-                ).repository_location.get_repository(attribute),
+                ).repository_location
+            )
+            yield (
+                instance,
+                workspace_context,
+                repository_location.get_repository(attribute),
             )
 
 
 @contextmanager
 def instance_with_multiple_repos_with_sensors(overrides=None):
     with instance_for_test(overrides) as instance:
         with create_test_daemon_workspace_context(
-            workspace_load_target(None), instance=instance
+            workspace_load_target(), instance=instance
```

Comment:
The `None` passed to `workspace_load_target` is required here. If no value is provided, the `workspace_load_target` tries to load a repo with the name `the_repo`. Since we're trying to load multiple repos here, we can't have a specific repo name specified. 

There's a couple of tests failing in BK right now because of it - test_cross_repo_run_status_sensor, test_instance_run_status_sensor , test_cross_repo_job_run_status_sensor,